pkg/summarizer/summary.go

```diff
@@ -531,7 +531,7 @@ func statusMessage(cols int, rows []*statepb.Row, recent int) string {
 const noGreens = "no recent greens"
 
 // latestGreen finds the ID for the most recent column with all passing rows.
-func latestGreen(grid *statepb.Grid) string {
+func latestGreen(grid *statepb.Grid, firstExtraColumn bool) string {
```

Comment:
Parameter name is still confusing to me, maybe something like 'useFirstExtra'?

Reply:
Fixed